handler/handler.go

```diff
@@ -3,29 +3,51 @@ package handler
 import (
 	"github.com/exlibris-fed/exlibris/activitypub"
 	"github.com/exlibris-fed/exlibris/config"
+	"github.com/exlibris-fed/exlibris/infrastructure/authors"
+	"github.com/exlibris-fed/exlibris/infrastructure/books"
+	"github.com/exlibris-fed/exlibris/infrastructure/covers"
+	"github.com/exlibris-fed/exlibris/infrastructure/reads"
+	"github.com/exlibris-fed/exlibris/infrastructure/registrationkeys"
+	"github.com/exlibris-fed/exlibris/infrastructure/reviews"
+	"github.com/exlibris-fed/exlibris/infrastructure/subjects"
+	"github.com/exlibris-fed/exlibris/infrastructure/users"
 	"github.com/exlibris-fed/exlibris/service"
 
 	"github.com/jinzhu/gorm"
 )
 
 // A Handler accepts http requests.
 type Handler struct {
-	db              *gorm.DB
-	ap              *activitypub.ActivityPub
-	cfg             *config.Config
-	bookService     *service.Book
-	authorService   *service.Author
-	editionsService *service.Editions
+	ap                   *activitypub.ActivityPub
+	cfg                  *config.Config
+	bookService          *service.Book
+	authorService        *service.Author
+	editionsService      *service.Editions
+	booksRepo            *books.Repository
```

Comment:
Do we need the `Repo` in each of these?

Reply:
no, I just wanted to convey it was a repository.